src/FakeItEasy/Expressions/ArgumentConstraints/EqualityArgumentConstraint.cs

```diff
@@ -30,7 +30,7 @@ public override string ToString()
             var stringValue = this.ExpectedValue as string;
             if (stringValue != null)
             {
-                return "\"{0}\"".FormatInvariant(stringValue);
+                return '"' + stringValue + '"';

```

Comment:
I see several places where we enclose a value between double quotes, using various approaches. Maybe it would be worth having an `DoubleQuote` extension method...